nvs.sh

```diff
@@ -157,7 +157,7 @@ nvs() {
 	# This allows the invocation to potentially modify the caller's environment (e.g. PATH)
 	if [ -f "${NVS_POSTSCRIPT}" ]; then
 		. "${NVS_POSTSCRIPT}"
-		rm "${NVS_POSTSCRIPT}"
+		rm -f "${NVS_POSTSCRIPT}"
```

Comment:
rm doesn’t prompt by default; the issue is that it’s using your local alias, which you’ve altered to do the prompt.
```suggestion
		command rm "${NVS_POSTSCRIPT}"
```

Reply:
Aah, of course:

```
❯ which rm
rm: aliased to nocorrect rm -i
```

This will do the trick. Thanks @ljharb!